README.md

```diff
@@ -216,223 +216,59 @@ interface StaticModuleRecord {
   // Static module records reflect their bindings for information only.
   // Loaders use internal slots for the compiled code and bindings.
   bindings: Array<Binding>;
+
+  // Indicates that initialize needs to receive a dynamic import function that
+  // closes over the referrer module specifier.
+  needsImport: boolean;
+
+  // Indicates that initialize needs to receive an importMeta.
+  needsImportMeta: boolean;
```

Comment:
These reflective properties go from “nice to have” to “essential” when the `Loader` is user code and can’t reach into internal slots.